pallets/pips/src/lib.rs

```diff
@@ -119,6 +124,21 @@ pub struct Pip<Proposal, Balance> {
     pub beneficiaries: Option<Vec<Beneficiary<Balance>>>,
 }
 
+/// A result of execution of get_votes.
+#[derive(Eq, PartialEq, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
+pub enum VoteCount<Balance> {
+    /// Proposal was found and has the following votes.
+    Success {
```

Comment:
Can we rename this to something like `ProposalFound`? `Success` can be confused with "proposal was accepted".